google/cloud/aiplatform/utils/yaml_utils.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-# Copyright 2020 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
nit: preserve the initial year, or leave it untouched.
https://opensource.google/documentation/reference/copyright#the_year